docs/apache-airflow-providers-airbyte/operators/airbyte.rst

```diff
@@ -0,0 +1,38 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _howto/operator:AirbyteTriggerSyncOperator:
+
+AirbyteTriggerSyncOperator
+==========================
+
+Use the :class:`~airflow.providers.airbyte.operators.AirbyteTriggerSyncOperator` to
+trigger an existing ConnectionId sync job in Airbyte.
+
+WARNING: This operator triggers a synchronization job in Airbyte.
```

Comment:
@turbaszek I wrote this comment about the problem with idempotency. What do you think?

Reply:
Is there a possibility to reattach to already running job?

Reply:
After triggering the job using the `connectionId` at the moment it is only possible to follow the status of the job in progress using the `/jobs/get`. Trigger the Operator again will queue a new synchronization job with a new `job_id`.

Reply:
I think it would be best to make `job_id` optional and generate is as an `uuid` if not passed by user - in this way users won't have to care about uniqueness of `job_id`, WDYT?